module4-authentication-and-security/r1.1.1-authentication-persistence/README.md

```diff
@@ -0,0 +1,268 @@
+# Authentication Persistence
+
+In this lesson, we will tackle how to persist user login, or registration between requests. Learning objectives are:
+
+- Difference between stateful and stateless persistence
+- Implementing stateful persistence using express-sessions
+- Implementing stateless identifiers using JSON Web Tokens (JWT)
+
+## **Stateful Session Storage**
```

Comment:
dont need to bold headers

Reply:
I've bolded all headers because I felt they weren't rendered with the needed hierarchy. But If they would look ok on GitHub or Canvas, happy to undo